### PR TITLE
postal 1.1.9 :snowflake:

### PR DESCRIPTION
**Pull request by ifitchet**
postal 1.1.9 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5230]
- dev_url:        https://github.com/openvenues/pypostal/tree/bf3762124e0b08231aa8f3d0cfee835403093002
- conda_forge:    https://github.com/conda-forge/postal-feedstock/blob/main/recipe/meta.yaml
- pypi:           https://pypi.org/project/postal/1.1.9
- pypi inspector: https://inspector.pypi.io/project/postal/1.1.9

### Explanation of changes:

- basic build
- tests which do address parsing and expansions so "must" be using `libpostal`

[PKG-5230]: https://anaconda.atlassian.net/browse/PKG-5230?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,6 +21,8 @@ requirements:
   host:
     - python
     - pip
+    - wheel
+    - setuptools
     - nose
     - six
```
Comment (lorepirri) [inline recipe/meta.yaml]: isn't `six` just for runtime?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -20,6 +21,8 @@ requirements:
   host:
     - python
     - pip
+    - wheel
+    - setuptools
     - nose
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
    - nose >=1.0
```

https://github.com/openvenues/pypostal/blob/bf3762124e0b08231aa8f3d0cfee835403093002/setup.py#L26

File: recipe/meta.yaml [hunk 3]
```diff
@@ -28,11 +31,19 @@ requirements:
     - libpostal
     - six
 test:
```
Comment (lorepirri) [inline recipe/meta.yaml:33]:
```suggestion

test:
```

one space between sections

**Review by lorepirri — APPROVED** [review]
nice job, `libpostal` seems to be tested by this one,
a couple of nitpicks

File: recipe/meta.yaml (hunk 1 above)
Comment (ifitchet) [reply recipe/meta.yaml]: Hmm, that was in the original recipe.  I clearly didn't check.

File: recipe/meta.yaml (hunk 2 above)
Comment (ifitchet) [reply recipe/meta.yaml]: _\<sigh\>_ I didn't check that either! 😆

**Review by danpetry — APPROVED** [review]

**Pull request merged**